src/util.ts

```diff
@@ -0,0 +1,22 @@
+/**
+ * Converts a value which might be a number or a bigint into a number to be used with apis that don't support bigint.
+ *
+ * Throws an UnsafeConversionError if the conversion would result in an unsafe integer for the Number type
+ * @param value
+ */
+export const toNum = (value: number | bigint) => {
```

Comment:
@tristanmenzel should this be `toNumber` instead?

Reply:
good catch